tests/docker/alpine-apache2-fpm/Dockerfile

```diff
@@ -7,9 +7,13 @@ FROM php:$PHP_VERSION-fpm-alpine$ALPINE_VERSION
 
 RUN apk add --no-cache apache2 apache2-proxy libexecinfo bash
 
-ARG TRACER_VERSION
+RUN curl -Lf -o datadog-php-tracer-1.0.0-nightly.x86_64.tar.gz https://output.circle-artifacts.com/output/job/524b8724-5d23-47d9-ba2c-c134bd4a0ed1/artifacts/0/datadog-php-tracer-1.0.0-nightly.x86_64.tar.gz
 RUN curl -Lf -o /tmp/dd-library-php-setup.php https://raw.githubusercontent.com/DataDog/dd-appsec-php/installer/dd-library-php-setup.php && \
-    PHP_INI_SCAN_DIR=/etc/php/ php /tmp/dd-library-php-setup.php --no-appsec --tracer-version $TRACER_VERSION --php-bin all
+    PHP_INI_SCAN_DIR=/etc/php/ php /tmp/dd-library-php-setup.php --no-appsec --tracer-file datadog-php-tracer-1.0.0-nightly.x86_64.tar.gz --php-bin all
+
+#ARG TRACER_VERSION
```

Comment:
Not sure if you left this on purpose. Same on other dockefiles

Reply:
It's commented out because once the 0.84.0 release is out, the dockerfiles will be reverted.